Build style handles building and installing aerc binary.
Makefile should only install remaining files.
--- a/Makefile	2023-01-04 16:38:38.000000000 +0100
+++ b/Makefile	2023-01-06 20:23:16.175385382 +0100
@@ -106,11 +106,10 @@
 clean:
 	$(RM) $(DOCS) aerc
 
-install: $(DOCS) aerc wrap
+install: $(DOCS) wrap
 	mkdir -m755 -p $(DESTDIR)$(BINDIR) $(DESTDIR)$(MANDIR)/man1 $(DESTDIR)$(MANDIR)/man5 $(DESTDIR)$(MANDIR)/man7 \
 		$(DESTDIR)$(SHAREDIR) $(DESTDIR)$(SHAREDIR)/filters $(DESTDIR)$(SHAREDIR)/templates $(DESTDIR)$(SHAREDIR)/stylesets \
 		$(DESTDIR)$(PREFIX)/share/applications
-	install -m755 aerc $(DESTDIR)$(BINDIR)/aerc
 	install -m644 aerc.1 $(DESTDIR)$(MANDIR)/man1/aerc.1
 	install -m644 aerc-search.1 $(DESTDIR)$(MANDIR)/man1/aerc-search.1
 	install -m644 aerc-accounts.5 $(DESTDIR)$(MANDIR)/man5/aerc-accounts.5
